docs/dev/api.rst

```diff
@@ -30,24 +29,38 @@ API Calls
 Rate limiting
 ^^^^^^^^^^^^^
 
-The ListenBrainz API is rate limited via the use of rate limiting headers that are sent as part of
-the HTTP response headers. Each call will include the following headers:
+The ListenBrainz API is rate limited via the use of rate limiting headers that
+are sent as part of the HTTP response headers. Each call will include the
+following headers:
+
+- **X-RateLimit-Limit**: Number of requests allowed in given time window
+
+- **X-RateLimit-Remaining**: Number of requests remaining in current time
+  window
 
-* **X-RateLimit-Limit**: The number of requests allowed in a given time window.
-* **X-RateLimit-Remaining**: The number of requests remaining in the current time window.
-* **X-RateLimit-Reset-In**: The number of seconds when the current time window expires. We recommend you use this header, since it is resilient against clients with incorrect clocks.
-* **X-RateLimit-Reset**: The UNIX epoch number of seconds (without timezone) when the current time window expires. This is provided for compatibility with other APIs, but we recommend using the X-Ratelimit-Reset-In header.
+- **X-RateLimit-Reset-In**: Number of seconds when current time window expires
+  [#]_
 
-The rate limiting is automatic and the client must use these headers to determine the rate at which calls to the API can be made.
-If the client exceeds the number of requests allowed, the server will respond with error code **429** *Too Many Requests*. Requests that provide
-the *Authorization* header with a valid user token may receive higher rate limits than those without valid user tokens.
+- **X-RateLimit-Reset**: UNIX epoch number of seconds (without timezone) when
+  current time window expires [#]_
 
+Rate limiting is automatic and the client must use these headers to determine
+the rate to make API calls. If the client exceeds the number of requests
+allowed, the server will respond with error code ``429: Too Many Requests``.
+Requests that provide the *Authorization* header with a valid user token may
+receive higher rate limits than those without valid user tokens.
+
+.. [#] **Recommended**: This header is resilient against clients with incorrect
+   clocks
+.. [#] Provided for compatibility with other APIs, but we still recommend using
```

Comment:
I think this "best practices tip" is not visible enough anymore. Can we put it into the paragraph above?

Reply:
@mayhem Will grab it now. 😃